src/main/webapp/scripts/comments.js

```diff
@@ -0,0 +1,25 @@
+export { makeApiCall, buildRequestBody };
+
+function makeApiCall(videoId, commentText, successfulApiCallback, failedApiCallback) {
```

Comment:
Needs more descriptive method name. `postVideoComment` or something like that.

Reply:
Done